### PR TITLE
5344: documentation of bytea data type

### PR DESCRIPTION
**Pull request by msudberg**
This change is [Reviewable](https://reviewable.io/reviews/materializeinc/materialize/5731)

### CODE REVIEW

File: doc/user/content/sql/types/bytea.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+92  | backslash | `'\\' or '\134'` | `'\\'::bytea` | `\x5c`
+0 to 31 and 127 to 255  | "non-printable" octets | `'\xxx'` (octal value) | `'\001'::bytea` | `\x01`
+
+## Details
+
+### Valid casts
+
+#### From `bytea`
+
+You can [cast](../../functions/cast) `bytea` to [text](../text). Casts from bytea
```
Comment (sploiselle) [inline doc/user/content/sql/types/bytea.md:52]: s/Casts from bytea/Casts from `bytea`/

**Review by sploiselle — APPROVED** [review]
Looks great to me

**Review by benesch — APPROVED** [review]
LGTM too! 🎉

File: doc/user/content/sql/types/bytea.md (hunk 1 above)
Comment (msudberg) [reply doc/user/content/sql/types/bytea.md]: fixed

**Pull request merged**